### ISSUE TITLE
why traefik change user_agent(null)?

### ISSUE THREAD

**Issue by gouki777** [containous/traefik#5621]
### Do you want to request a *feature* or report a *bug*?

Bug

### Did you try using a 1.7.x configuration for the version 2.0?

- [ ] Yes
- [√] No

### What did you do?
I access nginx through traefik.
UA="curl/7.29.0" is GOOD.

but change UA=null.i found something wrong with UA.    
UA="Go-http-client/1.1"

I want to see the original message.
I  think traefik is wrong to modify ua.
```
curl -i local.test.com
{ "@timestamp": "2019-10-10T10:53:17","status": "200", "http_request": "http://local.test.com/", "method": "GET", "user_agent":"curl/7.29.0" }
curl -i local.test.com -A ""
{ "@timestamp": "2019-10-10T10:57:55","status": "200", "http_request": "http://local.test.com/", "method": "GET", "user_agent":"Go-http-client/1.1" }
```
### What did you expect to see?
I want to see the original message.
If the user requests null,Please pass UA=null to bankends

### Output of `traefik version`: (_What version of Traefik are you using?_)
traefik: V1.7.17
k8s:   v1.13.8

**Comment by gouki777** [comment]
kind/bug/confirmed

**Comment by traefiker** [comment]
Hi! I'm Træfiker :robot: the bot in charge of tidying up the issues.

I have to close this one because of its lack of activity :disappointed:

Feel free to re-open it or join our [Community Forum](https://community.containo.us/).

**Comment by sh7dm** [comment]
Confirmed to occur on v2.1.0-rc3 too.

**Comment by traefiker** [comment]
Closed by #6030.
